lib/rubocop/config.rb

```diff
@@ -80,7 +80,7 @@ def cop_enabled?(cop)
     def warn_unless_valid
       validate
     rescue Config::ValidationError => e
-      warn "Warning: #{e.message}".color(:red)
+      warn Rainbow.new.wrap("Warning: #{e.message}").red
```

Comment:
What is the difference between `Rainbow("some string").red` and `Rainbow.new.wrap("some string").red`?